test/setup.ts

```diff
@@ -1,6 +1,4 @@
 import type { Jest } from '@jest/environment';
-// Check for missing or pending http mocks
-import './http-mock';
```

Comment:
revert

Reply:
sorry, completely overseen that previously was at the beginning of the file...